v-next/hardhat-ethers/test-tmp/helpers/artifacts/ambiguous-library.ts

```diff
@@ -0,0 +1,14 @@
+import type { Artifact } from "@ignored/hardhat-vnext/types/artifacts";
```

Comment:
These are the mocked artifacts mentioned in the PR description that currently do not have the source code that generates them, as discussed during the team meeting sync